src/components/ReviewsPage.jsx

```diff
@@ -0,0 +1,27 @@
+import { useState, useEffect } from "react"
+import ReviewCards from "./ReviewCards"
+
+const ReviewsPage = () => {
+
+    const [isLoading, setIsLoading] = useState(true);
+    const [reviews, setReviews] = useState([]);
+    const [category, setCategory] = useState("")
+
+    useEffect(() => {
+        fetch("https://nc-games-portfolio-project.onrender.com/api/reviews")

```

Comment:
This should be extracted as a function into a separate api.js file and then imported in as you need it. This file will then hold all your api functions across the entire app